lib/DB/ProjectIRDB.cpp

```diff
@@ -266,6 +268,10 @@ void ProjectIRDB::print() const {
   }
 }
 
+void ProjectIRDB::printAsJson(llvm::raw_ostream &OS) const {
+  OS << StatsJson.dump();
```

Comment:
```suggestion
  OS << StatsJson.dump(4) << '\n';
```

Otherwise, it may not format it properly